ktxspec.adoc

```diff
@@ -879,59 +879,110 @@ standard gamut images.
-or without first conditioning the data with Rate Distortion
-Optimization. If supercompression is used, the DFD's `bytesPlane[0-7]`
-must be set to 0 as described in the next subsection.
-
+Basis Universal UASTC Format::
+The Universal ASTC image format (UASTC) is indicated by `colorModel`
+`KDF_DF_MODEL_UASTC` (= 166) together with `vkFormat` `VK_FORMAT_UNDEFINED`
+(= 0). The DFD must be as described in Section 5.6.14 "KHR_DF_MODEL_UASTC"
+of <<KDF13>>. Images in this format must be transcoded to a
+GPU-supported block-compressed format before being uploaded to and
+sampled by a GPU. UASTC images can be supercompressed with Zstandard
+(`supercompressionScheme` = 2) with or without first conditioning
+the data with Rate Distortion Optimization. If supercompression is
+used, the DFD's `bytesPlane[0-7]` must be set to 0 as described in
+the next subsection.
++
+This color model provides channel Ids, e.g. `KHR_DF_CHANNEL_UASTC_RGB`
+that must be used to indicate the effective number of components
+in the data.  Consumers use this information to help select a
+transcode target. The id to use depends on the pre-deflation
+components as detailed below.
++
+_One-component images_ have the component value replicated in all 3
+channels of the rgb portion of the compressed data.
+`KHR_DF_CHANNEL_UASTC_RRR`.
++
+_Two-component images_ must have the R component replicated in all 3
+channels of the rgb portion and the G component in the alpha channel.
+`KHR_DF_CHANNEL_UASTC_RRRG`.
++
+_Three-component images_ must have their RGB components in the rgb
+channels. `KHR_DF_CHANNEL_UASTC_RGB`.
++
+_Four-component images_ must have their RGB components in the rgb
+channels and the alpha component in the alpha channel.
+`KHR_DF_CHANNEL_UASTC_RGBA`.
++
 The format of the UASTC data is described in Chapter 25 "UASTC
 Compressed Texture Image Format" of <<KDF13>>.
 
-===== DFD for Supercompressed Data
-When `<<_supercompressionscheme,supercompressionScheme>> is not 0`
-the _dfDescriptorBlock_ must preserve the `colorModel`, `transferFunction`,
-`colorPrimaries`, and `flags` of the pre-deflation images - for
-BasisLZ (`supercompressionScheme` = 1) the pre-encoding and
--deflation images - along with each sample's `channelType` and
-qualifiers in order to indicate which components are present in the
-deflated images. `bytesPlane[0-7]` must be set to 0 to indicate an
-_unsized format_, as described in Section 5.19 "Unsized Formats"
-of <<KDF13>>.
-
-For BasisLZ, `texelBlockDimension[0-3]` and each sample's
-`bitlength`, `bitOffset`, `sampleLower` and `sampleUpper` must be
-0
+[[etc1s]]
+Basis Universal ETC1S Format::
+The ETC1S image format is indicated by `colorModel` `KDF_DF_MODEL_ETC1S`
+(= 163) together with `vkFormat` `VK_FORMAT_UNDEFINED` (= 0). The
+DFD must be as described in Section 5.6.11 "KHR_DF_MODEL_ETC1S" of
+<<KDF13>>. Because ETC1S does not support an alpha component, Basis
+Universal uses 2 _slices_, (_planes_ in DFD-speak).
```

Comment:
Planes or samples? Also see below.

Reply:
Slices correspond to planes. Samples are more general. I think this is fine.